clearpath_control/launch/teleop_joy.launch.py

```diff
@@ -90,16 +98,45 @@ def generate_launch_description():
         name='teleop_twist_joy_node',
         parameters=[
             config_teleop_joy,
-            {'use_sim_time': use_sim_time}],
+            {'use_sim_time': use_sim_time},
+            {'publish_stamped_twist': True},
+        ],
         remappings=[
             ('joy', 'joy_teleop/joy'),
-            ('cmd_vel', 'joy_teleop/cmd_vel'),
+            ('cmd_vel', 'joy_teleop/_/cmd_vel_src'),
```

Comment:
Can we use `joy_teleop/_cmd_vel_src` to match our other hidden topic naming conventions?

For example: https://github.com/clearpathrobotics/clearpath_robot/blob/d240de9a0b058d130a870122e8100d6ab368b2ab/clearpath_hardware_interfaces/src/lighting/lighting.cpp#L204

Reply:
Maybe just call it `joy_teleop/_cmd_vel`

Reply:
I can rename the topic, no problem.

Reply:
Done.